packages/rest/src/validation/request-body.validator.ts

```diff
@@ -145,10 +152,10 @@ async function validateValueAgainstSchema(
 
   let validationErrors: ajv.ErrorObject[] = [];
   try {
-    const validationResult = await validate(body);
-    // When body is optional & values is empty / null, ajv returns null
+    const validationResult = await validate(value);
+    // When value is optional & values is empty / null, ajv returns null
     if (validationResult || validationResult === null) {
-      debug('Request body passed AJV validation.');
+      debug('Value passed AJV validation.');
```

Comment:
We can use `options.position` for debugging too.

Reply:
Good point. Added.